### PR TITLE
Importer structure

### PR DESCRIPTION
**Pull request by roald-teunissen**
An initial draft/proposal for a new structure for the importers.
[See the following issue for more information](https://github.com/thegreenwebfoundation/admin-portal/issues/249)

### CODE REVIEW

**Comment by mrchrisadams** [comment]
This is really nice @roald-teunissen , thanks.

I've added a bit of feedback, but once you have made those changes, please merge it in - there's no need to wait for me to give a second review.

**Review by mrchrisadams — CHANGES_REQUESTED** [review]
Hi Roald, this is good the changes are mainly stylistic, and about file placement - the actualy code I'm happy with. Danke!

File: apps/greencheck/management/commands/importer_interface.py
Comment (roald-teunissen) [reply apps/greencheck/management/commands/importer_interface.py:58]: In this case, we only return something when it is successfully created. Otherwise we return nothing. Do you want me to return None in that case?
By the way, what can I set as a return type, GreencheckASN?

File: apps/greencheck/management/commands/equinix_importer.py
Comment (roald-teunissen) [reply apps/greencheck/management/commands/equinix_importer.py:12]: Oh, I haven't thought of that. I thought the _providername_importer.py_ was a replacement for update_providername_ip_ranges.py. We do can split these two so we have a command file and a specification for the importer itself.

Comment (roald-teunissen) [reply apps/greencheck/management/commands/equinix_importer.py:12]: What do you think of renaming _update_equinix_ip_ranges_(.py) to one of these:
- _update_equinix_network_
- _update_equinix_ranges_
- _update_equinix_in_db_

File: apps/greencheck/management/commands/equinix_importer.py
Comment (roald-teunissen) [reply apps/greencheck/management/commands/equinix_importer.py:46]: I plan on doing that in the importer interface, yes.

File: apps/greencheck/management/commands/equinix_importer.py
Comment (roald-teunissen) [inline apps/greencheck/management/commands/equinix_importer.py:39]: What is this used for and do we actually need it?

**Comment by roald-teunissen** [comment]
Since [my last commit](https://github.com/thegreenwebfoundation/admin-portal/pull/251/commits/761925789456adf1818144fd8391e78eb5534dca) I have moved the AWS importer to the new structure. Although I have left in the old command for AWS alongside the new one, mainly because conftest.py is reliant on the old AWS importer.

@mrchrisadams do you know how we can solve this to work with the new structure? Maybe we can remove it as it seems like an old test?

**Comment by roald-teunissen** [comment]
With Azure, we still use a locally stored JSON instead of an endpoint.[ I've created an issue for this.](https://github.com/thegreenwebfoundation/admin-portal/issues/253) 

The three major providers we've discussed are in this PR right now. Only the following needs to be done:
- Handle the output of a command at the end of tests [ ]
- Handle the output of importers [ ]
- Renaming of datasets and datasets [ ]
- Clean up comments [ ]

File: apps/greencheck/management/commands/equinix_importer.py
Comment (mrchrisadams) [reply apps/greencheck/management/commands/equinix_importer.py]: this is the infomation disclosed when you pass a `--help` flag to a command, and is shown when listing all the possible commands dango offers

File: apps/greencheck/management/commands/equinix_importer.py
Comment (roald-teunissen) [reply apps/greencheck/management/commands/equinix_importer.py]: winner: update_equinix_networks_in_db

**Review by mrchrisadams — APPROVED** [review]

**Pull request merged**
